app/models/grda_warehouse/hud/client.rb

```diff
@@ -454,7 +454,7 @@ def hmis_source_visible_by?(user)
     scope :race_native_hi_other_pacific, -> do
       where(
         id: GrdaWarehouse::WarehouseClient.joins(:source).
-          where(c_t[:NativeHIOtherPacific].eq(1)).
+          where(c_t[:NativeHIOtherPacific].eq(1).or(c_t[:NativeHIPacific].eq(1))).
```

Comment:
Put `TodoOrDie` here instead of the `or`? I'm concerned about leaving the old column name as a landmine if we remove it from the DB.

Reply:
Done.